cypress/integration/channel/header_spec.js

```diff
@@ -70,4 +70,28 @@ describe('Header', () => {
 
         cy.apiSaveMessageDisplayPreference();
     });
+    it('MM-13483 - Cleared search term should not reappear as RHS is opened and closed', () => {
```

Comment:
Please change test key.
```suggestion
    it('S13483 - Cleared search term should not reappear as RHS is opened and closed', () => {
```

Reply:
Done